src/domain/usecases/user-monitoring/user-groups-monitoring/MonitorUserGroupsUseCase.ts

```diff
@@ -75,7 +73,7 @@ export class MonitorUserGroupsUseCase {
         log.info(`Get user groups with ids: ${options.groupsToMonitor.join(", ")}`);
 
         const getGroupsUseCase = new GetUserGroupsUseCase(this.userGroupRepository);
-        const compareUserGroupsUseCase = new CompareUserGroupsUseCase();
+        const compareUserGroupsUseCase = new CompareUserGroups();
```

Comment:
we can now remove `UseCase` from the name

Reply:
Yeah, sorry about that...